### DISCUSSION TITLE
Focus is reset to the top of the page when a lot of different actions are taken (creating a highlight, highlighting a text to annotate it and then closing the fly out)

### DISCUSSION THREAD

**Discussion by rhoadsstevens** [ManifoldScholar/manifold#3475] [Accessibility]
Priority level 1 (out of 3) from UW Accessible Technologies

WCAG mapping: 3.2.1 On Focus

STEPS TO REPRODUCE:
1. Select a book (i.e. "I My Recall")
2. Turn on caret browsing (insert + F7)
3. Tab into the text
4. Arrow through the text, hold shift using arrow keys to highlight
5. Press "enter" to highlight, then watch the focus indicator location

OBSERVED BEHAVIOR
Focus is reset to the top of the page when a lot of different actions are taken (creating a highlight, highlighting a text to annotate it and then closing the fly out)

EXPECTED BEHAVIOR:
The focus should remain where you left off, that way people don't have to find their spot again.

**Comment by dananjohnson** [comment]
We addressed issues around focus management with annotating and highlighting in [v8](https://manifoldscholar.github.io/manifold-docusaurus/docs/administering/release_notes/v8x#keyboard-annotation).
